### DISCUSSION TITLE
Conventional Comments: worth trying

### DISCUSSION THREAD

**Discussion by yajo** [OCA/odoo-community.org#119] [Ideas]
I found this tool to improve intercommunication when doing code reviews: https://conventionalcomments.org/

IMHO applying that convention would help our community to collaborate better.

I'd love to test it in my upcoming reviews where needed. Maybe some more of you are interested in testing this standard too. After a few reviews we can see if it improves the contribution experience.

**Comment by legalsylvain** [comment]
Tried here : https://github.com/OCA/product-attribute/pull/1393#pullrequestreview-1680155903

**Comment by francesco-ooops** [comment]
@yajo any possibility to implement this in review process? as in, making it visible to anyone reviewing

**Reply by yajo** [reply]
I guess we should just open a PR for improvement against https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

**Comment by HekkiMelody** [comment]
Tried it multiple times in OCA, and I think it works very well! 

Introduced it to my company as well and everyone likes it, now we use it for internal reviewing as well. Thanks a lot for bringing it to our attention.

**Reply by yajo** [reply]
Same experience here, indeed.
